### PR TITLE
145/show sshkeys stored

### LINKED ISSUE #145: Show sshkeys stored

### PR DESCRIPTION
**Pull request by iLeonelPerea**
closes #145 

changes proposed in the pull request:

- Create sshKeyItem Component
- Show added sshkeys

@thetonymaster

### CODE REVIEW

File: src/components/provisionerForm/SSHKeys.js [hunk 1]
```diff
@@ -31,88 +49,110 @@ const SSHKeys = ( {deleteSSHKey, enableSSHKey, setSSHKey, setSSHKeyTitle, setSSH
...
+      sshKeysAppState.get('ssh_keys').map((value, index) => 
+        <SSHKeysItem
+          handleClick={handleSSHKeyClick}
+          key={index}
+          value={value.get('title')}
+          isActive={value.get('enable')?true:false}
+          end={(index == sshKeysAppState.get('ssh_keys').size - 1) ? "end" : ""}
+        />) : "";
+  
+  const AddSHHKey =
```
Comment (thetonymaster) [inline src/components/provisionerForm/SSHKeys.js]: Typo.

File: src/components/provisionerForm/SSHKeys.js [hunk 2]
```diff
@@ -31,88 +49,110 @@ const SSHKeys = ( {deleteSSHKey, enableSSHKey, setSSHKey, setSSHKeyTitle, setSSH
...
+        />) : "";
+  
+  const AddSHHKey = 
+    !(sshKeysAppState.get('show_ssh_key')) ?
+      <a
+        href="javascript:void(0);"
+        onClick={ShowSSHKeyKeypress}
+        id="show_ssh_key">+ Add SSH Key</a> : "";
+      
+  const ShowSSHKey =
```
Comment (thetonymaster) [inline src/components/provisionerForm/SSHKeys.js]: This seems like a rather large method, maybe can be in a component or something, up to you.

File: src/components/provisionerForm/SSHKeys.js [hunk 3]
```diff
@@ -31,88 +49,110 @@ const SSHKeys = ( {deleteSSHKey, enableSSHKey, setSSHKey, setSSHKeyTitle, setSSH
...
-                    rows="5"
-                    onChange={AddSSHKeyContent}
-                    placeholder="SSH Key Content"></textarea>
-                </label>
-                <div 
-                  className="hide" 
-                  id="ssh_key_content_value_error">
-                  <small className="error hide">SSH Key Content can't be blank</small>
-                </div>
+  const AddedSSHKeys =
```
Comment (thetonymaster) [inline src/components/provisionerForm/SSHKeys.js]: I just have a little nitpick with the name of the function, all the other functions have an action name, like add or show, added just seems rather weird, maybe change the function name? Something like **GetSSHKeys** or **PrinSSHKeys**, something like that.

File: src/components/provisionerForm/SSHKeys.js (hunk 2 above)
Comment (iLeonelPerea) [inline src/components/provisionerForm/SSHKeys.js]: SSHKeysItem is a subcomponent

File: src/components/provisionerForm/SSHKeys.js (hunk 3 above)
Comment (iLeonelPerea) [inline src/components/provisionerForm/SSHKeys.js]: yeah. let me change some functions names

**Comment by iLeonelPerea** [comment]
@thetonymaster looks better?

**Comment by thetonymaster** [comment]
LGTM

**Pull request merged**